pkg/llms/zhipuai/api.go

```diff
@@ -35,9 +35,10 @@ const (
 type Model string
 
 const (
-	ZhiPuAILite Model = "chatglm_lite"
-	ZhiPuAIStd  Model = "chatglm_std"
-	ZhiPuAIPro  Model = "chatglm_pro"
+	ZhiPuAILite      Model = "chatglm_lite"
+	ZhiPuAIStd       Model = "chatglm_std"
+	ZhiPuAIPro       Model = "chatglm_pro"
+	ZhiPuAIEmbedding Model = "text_embedding"
```

Comment:
should  we treat `ZhiPuAIEmbedding`   as a model?

Reply:
It is used in `BuildAPIURL` while generating post request, since ZhiPuAI official embedding api requires post request to call this name.